### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by ioanniskousis**
#  Restaurant Page

### - Since webpack is used, run 'npm run build' on you terminal before opening

- The project implements an HTML page - the index.html - rendered using JavaScript commands.  
  Webpack is used having configured as follows  
    - entry  : /src/index.js  
    - output : dist/main.js  
    - Images are compiled using file-loader  
    - CSS is compiled using style-loader  

- The mark-up file dist/index.html only contains a skeleton that is rendered with elements depending on which view is selected by the user between 3.
    1. About Us, rendered by aboutUs.js  
    2. Menu, rendered by menu.js fed with data from menuItems.js  
    3. Contact, rendered by contacts.js  

  The page **header**, page **footer**, and the container divisions (**main** and **mainContent**) of the **content** are rendered by the skeleton.js  
  The navigation division is rendered by navigator.js  

- The navigation buttons are supplied a pointer to a callback function (the navigatorCallBack function) from the index.js and this is called when a button is clicked providing the index of the button as a parameter.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<body>
+  <header>
+    <div id="header-caption"></div>
+  </header>
+  <main>
+    <div id="mainContent">
+      <div id="content"></div>
+    </div>
+  </main>
+  <footer id="footer"></footer>
```
Comment (bellom) [inline dist/index.html]: Awesome job so far.
we need to change the structure of this file.

Well done developer.... 👍 
According to the instruction on `Odin` page, we should only have `<div id="content"></div>` in side the `body` tag. so you need to remove the `header`, `the extra div`, and `footer` tags added.
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (bellom) [inline package.json:8]: We should have all necessarily used packages that should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) **in our case we are missing `webpack-dev-server`**
Kindly add `webpack-dev-server`.

File: src/menu.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  content.appendChild(saladsHeader);
+}
+
+function renderItem(content, item) {
+  const menuItem = crel('div');
+  menuItem.className = 'menuItem';
+
+  const itemImage = crel('div');
+  itemImage.className = 'itemImage '.concat(item.image);
+  // itemImage.style.backgroundImage = 'url("../resources/images/'.concat(item.image).concat('")');
```
Comment (bellom) [inline src/menu.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ioanniskousis  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/menu.js (hunk 3 above)
Comment (ioanniskousis) [reply src/menu.js]: Thank you Oluwaseun for reviewing my request.  
I did all changes you suggested.

**Review by shloch — APPROVED** [review]
## Hello @ioanniskousis 

Great job with the corrections made
resto

App still works and is stable
Thumbs up, you can merge this

**Pull request merged**